lib/ProductOpener/Users.pm

```diff
@@ -380,6 +380,9 @@ sub check_user_form($$$) {
 		elsif ($user_ref->{userid} !~ /^[a-z0-9]+[a-z0-9\-]*[a-z0-9]+$/) {
 			push @{$errors_ref}, $Lang{error_invalid_username}{$lang};
 		}
+		elsif (length($user_ref->{userid}) > 20) {
+			push @{$errors_ref}, $Lang{error_username_too_long}{$lang};
+		}		
```

Comment:
Being at it, can't we also add a check on username length < 60 characters for example (to avoid spam)

Reply:
@alexgarel I added the check.